src/handlers/dispersion.ts

```diff
@@ -81,3 +81,25 @@ export const getTrendingAddresses = async (): Promise<
     return null;
   }
 };
+
+export const getAdditionalAssetData = async (address: EthereumAddress) => {
+  try {
+    const res = await dispersionApi.get(`/dispersion/v1/expanded/1/${address}`);
+    return res?.data?.data;
```

Comment:
let's follow the similar pattern of explicitly return `?? null` here and `null` in the catch